lib/game_box_web/live/arena_live.ex

```diff
@@ -74,7 +73,7 @@ defmodule GameBoxWeb.ArenaLive do
             <div>
               <p>
                 <strong>Player count: </strong>
-                <%= @total_players %>-<%= @constraints.min_players %>
+                <%= @total_players %>-<%= assigns[:constraints][:min_players] %>
```

Comment:
keys not guaranteed to exist? the _why_ might be clearer with `get_in()` but probably doesn't matter